xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.h

```diff
@@ -76,7 +77,12 @@ class CDVDInputStreamNavigator
   int GetTotalButtons() override;
   bool GetCurrentButtonInfo(CDVDOverlaySpu* pOverlayPicture, CDVDDemuxSPU* pSPU, int iButtonType /* 0 = selection, 1 = action (clicked)*/);
 
-  bool HasMenu() override { return true; }
+  /*!
+   * \brief Get the supported menu type
+   * \return The supported menu type
+  */
+  MenuType GetSupportedMenuType() override { return MenuType::NONE; }
```

Comment:
is it correct to change from `HasMenu() -> true` to `GetSupportedMenuType() -> MenuType::NONE` ?

Reply:
Thanks for noticing, it's an oversight from my side and would break dvd menus!